src/libs/ReportActionsUtils.ts

```diff
@@ -1728,13 +1729,24 @@ function isCardIssuedAction(reportAction: OnyxEntry<ReportAction>) {
     return isActionOfType(reportAction, CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED, CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED_VIRTUAL, CONST.REPORT.ACTIONS.TYPE.CARD_MISSING_ADDRESS);
 }
 
-function getCardIssuedMessage(reportAction: OnyxEntry<ReportAction>, shouldRenderHTML = false) {
-    const assigneeAccountID = (getOriginalMessage(reportAction) as IssueNewCardOriginalMessage)?.assigneeAccountID;
+function getCardIssuedMessage(reportAction: OnyxEntry<ReportAction>, shouldRenderHTML = false, policyID = '-1') {
+    const cardIssuedActionOriginalMessage = isActionOfType(
+        reportAction,
+        CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED,
+        CONST.REPORT.ACTIONS.TYPE.CARD_ISSUED_VIRTUAL,
+        CONST.REPORT.ACTIONS.TYPE.CARD_MISSING_ADDRESS,
+    )
+        ? getOriginalMessage(reportAction)
+        : undefined;
+
+    const assigneeAccountID = cardIssuedActionOriginalMessage?.assigneeAccountID ?? -1;
+    const cardID = cardIssuedActionOriginalMessage?.cardID ?? -1;
     const assigneeDetails = PersonalDetailsUtils.getPersonalDetailsByIDs([assigneeAccountID], currentUserAccountID ?? -1).at(0);
-
+    const isPolicyAdmin = PolicyUtils.isPolicyAdmin(PolicyUtils.getPolicy(policyID));
```

Comment:
I think this is wrong if policyID is -1, no?

Reply:
Yep, ```isPolicyAdmin``` will be ```false``` if policyID is -1

Reply:
This sounds wrong, because `-1` is not checked here https://github.com/Expensify/App/blob/9e1f21ac6c354988b45edfa88eb0950b240949c7/src/libs/PolicyUtils.ts#L579 😕 
But also checking with the team about the -1 reocommendation

Reply:
> This sounds wrong, because -1 is not checked here


- If policyID = -1, then condition:

https://github.com/Expensify/App/blob/9e1f21ac6c354988b45edfa88eb0950b240949c7/src/libs/PolicyUtils.ts#L579
is false

so undefined will be returned because of:

https://github.com/Expensify/App/blob/9e1f21ac6c354988b45edfa88eb0950b240949c7/src/libs/PolicyUtils.ts#L582

I think ```getPolicy(-1) = undefined``` makes sense.

> But also checking with the team about the -1 recommendation you linked above


- Waiting for the final decision